pkg/metallb/frrnodestatelist.go

```diff
@@ -0,0 +1,74 @@
+package metallb
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/schemes/metallb/frrtypes"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// ListFrrNodeState returns frr node state inventory in the given namespace.
+func ListFrrNodeState(
+	apiClient *clients.Settings, nsname string, options ...client.ListOptions) ([]*FrrNodeStateBuilder, error) {
```

Comment:
Please remove the nsname parameter and update the comment

Reply:
removed